config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: gcr.io/k8s-staging-kmm/kernel-module-management
-  newTag: main
+  newName: quay.io/ebelarte/kmmo
```

Comment:
I guess this is only a temporary change?

Reply:
Sure. Just for my tests. I'll remove it.